### PR TITLE
Bug 2033751: Rebase 1.23.0

### PR DESCRIPTION
**Pull request by soltysh**
Picked from https://github.com/openshift/origin/pull/26711

### CODE REVIEW

**Comment by deads2k** [comment]
/lgtm

be green!

**Comment by deads2k** [comment]
/lgtm cancel

I can't read the regex.  I'll either need a test or someone who is regex aware to lgtm it.

File: cmd/openshift-tests/images.go [hunk 1]
```diff
@@ -229,9 +230,22 @@ func pulledInvalidImages(fromRepository string) ginkgo.JUnitForEventsFunc {
...
 		allowedPrefixes := allowedPrefixes.List()
 
+		// these are the possible format we need to work with:
+		// 1. reason/Pulled Container image "quay.io/openshift/community-e2e-images:e2e-7-k8s-gcr-io-e2e-test-images-busybox-1-29-2-cqcP1Tnbm-JjJyUA" already present on machine
+		// 2. reason/Pulled image/quay.io/openshift/community-e2e-images:e2e-7-k8s-gcr-io-e2e-test-images-busybox-1-29-2-cqcP1Tnbm-JjJyUA
+		// the regexp tries to match either image/(group) or image "(group)",
+		// where (group) is constructed from three subgroups divided with /
+		// where each has one or more characters from these:
+		// \w (word characters - [0-9A-Za-z_]), -, :, . (needs escaping)
+		imageRe, err := regexp.Compile(`image/([\w-:/\.]+/[\w-:/\.]+/[\w-:/\.]+)$|image "([\w-:/\.]+/[\w-:/\.]+/[\w-:/\.]+)"`)
```
Comment (bparees) [inline cmd/openshift-tests/images.go:240]: why is `/` in the character class?

i'd expect `[\w-:\.]+`

**Comment by jsafrane** [comment]
/test e2e-aws-csi
I'm just curious

File: cmd/openshift-tests/images.go (hunk 1 above)
Comment (soltysh) [reply cmd/openshift-tests/images.go:240]: Good catch, it should've been removed when switching to three subgroups splitted with `/` - updating.

**Comment by vrutkovs** [comment]
sigh, I broke -serial test, see https://github.com/openshift/origin/pull/26765

**Comment by vrutkovs** [comment]
https://github.com/openshift/release/pull/25385 would fix TargetDown for promtail replicas

**Comment by wking** [comment]
[gcp][1]:

```
: [sig-arch][Early] Managed cluster should start all core operators [Skipped:Disconnected] [Suite:openshift/conformance/parallel]
--
Run #0: Failed 1s
fail [github.com/onsi/ginkgo@v4.7.0-origin.0+incompatible/internal/leafnodes/runner.go:113]: Jan 19 04:44:27.207: Some cluster operators are not ready: etcd (Degraded=True DefragController_Error: DefragControllerDegraded: cluster is unhealthy: 3 of 4 members are available, ci-op-419nnzbd-2a78c-dkjj9-master-2 is unhealthy)
```

Sounds similar to [rhbz#2040533][2], except in the bug the failing is at bootstrap time, but in this run the failure was late in the origin suite.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26753/pull-ci-openshift-origin-master-e2e-gcp/1483652197836132352
[2]: https://bugzilla.redhat.com/show_bug.cgi?id=2040533

**Comment by wking** [comment]
[e2e-aws-fips][1] looks like installer Terraform vs. AWS eventual consistency.

/retest

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26753/pull-ci-openshift-origin-master-e2e-aws-fips/1483652197727080448

**Comment by vrutkovs** [comment]
`alert/etcdHighNumberOfLeaderChanges` still plagues gcp upgrade :/

**Pull request merged**